models/pendo__feature_daily_metrics.sql

```diff
@@ -0,0 +1,73 @@
+with spine as (
+
+    select *
+    from {{ ref('int_pendo__calendar_spine') }}
+),
+
+daily_metrics as (
+
+    select *
+    from {{ ref('int_pendo__feature_daily_metrics') }}
+),
+
+feature as (
+
+    select 
+        *,
+        cast( {{ dbt_utils.date_trunc('day', 'created_at') }} as date) as created_on,
+        cast( {{ dbt_utils.date_trunc('day', 'last_click_at') }} as date) as last_click_on
+
+    from {{ ref('pendo__feature') }}
+),
+
+feature_spine as (
+
+    select 
+        spine.date_day,
+        feature.feature_id,
+        feature.feature_name,
+        feature.group_id, -- product_area ID
+        feature.product_area_name,
+        feature.page_id,
+        feature.page_name
+    
+    from spine 
+    join feature
+        on spine.date_day >= feature.created_on
+        and spine.date_day <= cast( {{ ['feature.valid_through', 'feature.last_click_on'] | max }} as date) -- or should this just go up to today?
```

Comment:
Zendesk is currently built to go to today. While I see the benefit, I also see the downside. Going to today will continue to generate data until the end of time even if there is no point. I like your approach of just focusing on the last click on date as the cap for the spine.